src/ontology/mondo-edit.obo

```diff
@@ -545406,6 +545405,49 @@ xref: OMIM:169500 {source="MONDO:equivalentTo"}
 is_a: MONDO:0008215 {source="OMIM:169500", source="https://orcid.org/0000-0002-4142-7153"} ! adult-onset autosomal dominant demyelinating leukodystrophy
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8968" xsd:anyURI
 
+[Term]
+id: MONDO:0700296
+name: aflatoxicosis
+def: "A disease caused by consuming food or feed contaminated with aflatoxins." [MONDO:patterns/poisoning, PMID:32491713]
+intersection_of: MONDO:0029000 ! poisoning
+intersection_of: realized_in_response_to_stimulus CHEBI:22271 ! aflatoxin
```

Comment:
do we want to assert a superclass? we can do this later if needed

Reply:
Yes, let's do it later.